Sources/GRPCCore/Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,28 @@
+# ``GRPCCore``
+
+A gRPC library for Swift written natively in Swift.
+
+> 🚧 This module is part of gRPC Swift v2 which is under active development and in the pre-release
+> stage.
+
+## Package structure
+
+gRPC Swift is made up of a number of modules, each of which is documented separately. However this
+module – ``GRPCCore`` – includes higher level documentation such as tutorials. The following list
+contains products of this package:
+
+- ``GRPCCore`` contains core types and abstractions and is the 'base' module for the project.
+- `GRPCInProcessTransport` contains an implementation of an in-process transport.
+- `GRPCHTTP2TransportNIOPosix` provides client and server implementations of HTTP/2 transports built
+  on top of SwiftNIO's POSIX Sockets abstractions.
+- `GRPCHTTP2TransportNIOTransportServices` provides client and server implementations of HTTP/2
+  transports built on top of SwiftNIO's Network.framework abstraction, `NIOTransportServices`.
+- `GRPCProtobuf` provides serialization and deserialization components for `SwiftProtobuf`.
+
+## Topics
```

Comment:
Is this section supposed to be empty or do we want to remove it / add something to it?

Reply:
It's not empty, it has the "Getting involved" subsection. There will be more subsections (e.g. tutorials/articles) in the future too. 

`## Topics` is also magic: if you don't add it then DocC will add it and list all articles etc.
 underneath it (as well as API docs). 

If you add it manually it'll add in everything that you didn't explicitly add, so at the moment it'll look like this:

<img width="779" alt="Screenshot 2024-08-14 at 13 22 47" src="https://github.com/user-attachments/assets/eba181f2-e7be-4d6e-a8fa-92de5ba2be8f">